PWGHF/Core/SelectorCuts.h

```diff
@@ -405,10 +432,22 @@ static const std::vector<std::string> labelsPt = {
   "pT bin 8",
   "pT bin 9",
   "pT bin 10",
-  "pT bin 11"};
+  "pT bin 11",
+  "pT bin 12",
+  "pT bin 13",
+  "pT bin 14",
+  "pT bin 15",
+  "pT bin 16",
+  "pT bin 17",
+  "pT bin 18",
+  "pT bin 19",
+  "pT bin 20",
+  "pT bin 21",
+  "pT bin 22",
+  "pT bin 23"  };
 
 // column labels
-static const std::vector<std::string> labelsCutVar = {"deltaM", "pT Pi", "pT K", "decay length", "normalized decay length XY", "cos pointing angle", "cos pointing angle XY", "max normalized deltaIP"};
+static const std::vector<std::string> labelsCutVar = {"deltaM", "pT Pi", "pT K", "d0 Pi", "do K", "Sigma vtx", "sum d0^2", "cos pointing angle", "cos pointing angle xy", "decay length", "decay length xy", "normalized decay length XY", "max normalized deltaIP"};
```

Comment:
AFAIK `Sigma vtx` has no equivalent in the Run3 framework. Therefore, it cannot be used in the selections. Are you planning to implement it?

Reply:
OK, for the moment i'll remove...Thanks